crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -129,7 +128,7 @@ pub async fn exec(config: Config, args: &clap::ArgMatches) -> anyhow::Result<()>
             build::exec_with_argstring(config.clone(), project_path, build_options).await?
         };
         let spinner = indicatif::ProgressBar::new_spinner();
-        spinner.enable_steady_tick(60);
+        spinner.enable_steady_tick(std::time::Duration::from_millis(60));
```

Comment:
After some digging, it looks like this must have been due to a package version upgrade. Did we need to bump this package version in this PR for some reason?

Reply:
There's a function in it I was looking to use that wasn't in the version we had specified, I believe